### PR TITLE
sabnzbd-testing to 1.0.0 RC5

### PR DESCRIPTION
**Pull request by mhartvig**

### CODE REVIEW

File: spk/sabnzbd-testing/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = sabnzbd-testing
-SPK_VERS = 0.8.0Beta5
-SPK_REV = 4
+SPK_VERS = 1.0.0RC5
+SPK_REV = 6
```
Comment (Dr-Bean) [inline spk/sabnzbd-testing/Makefile]: `5` :)

**Comment by Dr-Bean** [comment]
I assume there were no changes to config files needed? Did you test if upgrading from the current version works?

**Comment by mhartvig** [comment]
No config changes needed. I didn't test from 0.8.0 to 1.0.0 RC5, I tested from RC4 to RC5.

**Comment by GoodOmens83** [comment]
Might want to bump to 1.0 before merging.

**Comment by mhartvig** [comment]
@GoodOmens83 seems like version 1.0.0 has been removed again due to a bug.

**Comment by mhartvig** [comment]
The 1.0 has been released for good now, will make an update to that instead.

**Pull request closed without merging**